jenkins-pipelines/oss/longevity/longevity-schema-topology-changes-splitted-networks-12h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
```

Comment:
same comment as in other cases, let's not duplicate, but change the teir1 test to use it

Reply:
done